libdevcore/Common.h

```diff
@@ -33,6 +33,10 @@
 #define noexcept throw()
 #endif
 
+#ifdef __INTEL_COMPILER
+#pragma warning(disable:3682)
```

Comment:
Please could you add a comment to the right of the pragma with the warning message being suppressed?  Thanks!